api/src/main/java/org/pmiops/workbench/actionaudit/targetproperties/EgressEventTargetProperty.kt

```diff
@@ -0,0 +1,27 @@
+package org.pmiops.workbench.actionaudit.targetproperties
+
+import io.opencensus.common.Timestamp
+import org.pmiops.workbench.model.EgressEvent
+
+enum class EgressEventTargetProperty
+constructor(
+    override val propertyName: String,
+    override val extractor: (EgressEvent) -> String?
+) : ModelBackedTargetProperty<EgressEvent> {
+    EGRESS_MIB("egress_mib",
+            { it.egressMib?.toString() }),
+    TIME_WINDOW_START("time_window_start",
+            {
+                if (it.timeWindowStart != null)
```

Comment:
nit: IYI You can do this wiht a `.?` and `?:` chain. https://medium.com/tompee/idiomatic-kotlin-elvis-operator-d6f5639a5130